.github/workflows/scheduled_merge.yml

```diff
@@ -0,0 +1,114 @@
+name: Schedule Merge Staging into Main
+
+on:
+  workflow_dispatch:
+    inputs:
+      run_job:
+        description: 'Execute manually'
+        required: true
+        default: 'yes'
+  schedule:
+    - cron: '0 17 */15 * *'  # execute every 15 days at 17:00 UTC
+
+permissions: write-all
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.1
+
+      - name: Setup node
+        uses: actions/setup-node@v4.0.0
+        with:
+          node-version: '^18.0.0'
+          cache: 'npm'
+
+      - name: Install
+        run: npm ci
+
+      - uses: actions/cache@v3.3.2
+        with:
+          path: ./*
+          key: ${{ github.sha }}-${{ github.run_number }}
+  lint:
+    runs-on: ubuntu-latest
+    needs: install
+    steps:
+      - uses: actions/checkout@v4.1.1
+
+      - name: Setup node
+        uses: actions/setup-node@v4.0.0
+        with:
+          node-version: '^18.0.0'
+          cache: 'npm'
+
+      - uses: actions/cache@v3.3.2
+        with:
+          path: ./*
+          key: ${{ github.sha }}-${{ github.run_number }}
+
+      - run: npm run lint
+
+  build:
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v4.1.1
+
+      - name: Setup node
+        uses: actions/setup-node@v4.0.0
+        with:
+          node-version: '^18.0.0'
+          cache: 'npm'
+
+      - uses: actions/cache@v3.3.2
+        with:
+          path: ./*
+          key: ${{ github.sha }}-${{ github.run_number }}
+      
+      - name: Build
+        run: npm run build --if-present
+        env:
+          NEXT_PUBLIC_SANITY_PROJECT_ID: ${{ secrets.SANITY_PROJECT_ID }}
+          NEXT_PUBLIC_SANITY_DATASET: ${{ secrets.SANITY_DATASET }}
+
+  merge:
+    runs-on: ubuntu-latest
+    needs: build
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4.1.1
+      with:
+        persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal access token.
+        fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
+
+    - name: Configure user
+      run: |
+        git remote set-url origin "https://${{ secrets.PAT_TOKEN }}@github.com/funas-org/cv-tests.git"
```

Comment:
You can go ahead and add the `PAT_TOKEN` to Github actions secrets by logging into the cv-dev account, and then I think we also need to update the remote origin url.

Reply:
Ohh great catch! I forgot to change that